_Sidebar.md

```diff
@@ -74,6 +74,7 @@
     * [[Automated tests|Tests]]
       * [[TypeScript tests|TypeScript-tests]]
       * [[Backend tests|Backend-tests]]
+        * [[Testing email functionality|Testing-email-functionality]]
```

Comment:
Should this go in a separate section from "Automated tests", since it's a manual test? Maybe we can put it in the "Manual tests" section?

Reply:
Good idea, Thanks!